src/project_types/file_layout.rs

```diff
@@ -86,24 +90,24 @@ impl FileLayout {
             if channel.channelName.len() > 40 {
                 return Err(Error::InvalidLayout(String::from("Channel name cannot be longer than 40 characters")));
             }
-            if !channel.channelName.chars().all(|c| c.is_alphanumeric() || c == ' ') {
-                return Err(Error::InvalidLayout(String::from("Channel name has to be alphanumeric: ") + &channel.channelName));
+            if !channel.channelName.chars().all(|c| c.is_alphanumeric() || " -/_".contains(c)) {
```

Comment:
Out of curiosity, do we know if Vixen accepts a literal `\`?

Reply:
Not sure; these whitelists came out of what Josh organically used in channel names, and he didn't use a \

Reply:
Channel names can be changed to whatever you need.  Remaking the profile is
fairly easy

Reply:
Probably worth an experiment for later but don't concentrate on it.

Reply:
Remaking the profile may be easy, but it's an annoying and unnecessary extra step. The character restrictions were fairly arbitrary, so adding a few commonly used non-alphanumerics is fine. Just building up the whitelist :)